packages/clay-dropdown/src/ClayDropdownBase.soy

```diff
@@ -156,7 +162,7 @@
 		<div class="dropdown-section">
 			<div class="input-group input-group-sm">
 				<div class="input-group-item">
-					<input aria-label="Search for..." class="form-control input-group-inset input-group-inset-after" data-oninput="{$_handleSearch}" placeholder="Search for..." type="text" ref="searchInput" />
+					<input aria-label="{$ariaLabel}" class="form-control input-group-inset input-group-inset-after" data-oninput="{$_handleSearch}" placeholder="{$placeholder}" type="text" ref="searchInput" />
```

Comment:
Hey @antonio-ortega instead of exposing an API to configure we can use the `{msg desc=""}search-for{/msg}`? We usually did it in other places.

Reply:
That make complete sense. My lack of knowledge made me send that silly way to localize these values. I've just forced-pushed my changes in this branch with your suggestions! Thanks @matuzalemsteles !

Reply:
@antonio-ortega do not worry!